src/luascript.cpp

```diff
@@ -4468,9 +4470,15 @@ int LuaScriptInterface::luaGameCreateMonster(lua_State* L)
 	const Position& position = getPosition(L, 2);
 	bool extended = getBoolean(L, 3, false);
 	bool force = getBoolean(L, 4, false);
-	if (g_game.placeCreature(monster, position, extended, force)) {
-		pushUserdata<Monster>(L, monster);
-		setMetatable(L, -1, "Monster");
+	bool result = g_events->eventMonsterOnSpawn(monster, position, false, true);
+	if (result) {
```

Comment:
The variable `result` is not necessary: `if (g_events->eventMonsterOnSpawn(monster, position, false, true)) {`

I am also debating whether it would be desirable to have the `force` parameter ignore or overwrite the event.
e.g.
`if (force || g_events->eventMonsterOnSpawn(monster, position, false, true)) {`
or
`if (g_events->eventMonsterOnSpawn(monster, position, false, true) || force) {`

Reply:
It's extra functionality, so sure?
Can't think of any scenarios where you would though.

Force a monster to spawn into the position of another?